docs/content/latest/api/ysql/pg-extensions.md

```diff
@@ -0,0 +1,56 @@
+---
+title: PostgreSQL extensions
+headerTitle: PostgreSQL extensions
+linkTitle: PostgreSQL extensions
+description: List of supported PostgreSQL extensions
+summary: Reference for YSQL extensions
+menu:
+  latest:
+    identifier: api-ysql-postgresql-extensions
+    parent: api-ysql
+    weight: 4400
+isTocNested: true
+showAsideToc: true
+---
+
+PostgreSQL provides a way to extend the functionality of a database by bundling SQL objects into a package and using them as a unit. 
+This page describes the PostgreSQL extensions and describes the extensions supported by YugabyteDB.
+
+
+# PostgreSQL extensions supported by YugabyteDB
+For information about using a specific extension, see the documentation link in one of the tables below.
```

Comment:
I believe we have pre-bundled extensions and then some that we can choose to install later. So the status must be on those lines. Also, the description for the rest of the extensions need to be completed.

Reply:
Done.